api/comment_threads.rb

```diff
@@ -78,6 +78,7 @@
   comment.anonymous_to_peers = bool_anonymous_to_peers || false
   comment.author = user
   comment.comment_thread = thread
+  comment.children_count = 0
```

Comment:
Yes.  With a word like "user", where "users" is the plural, the difference doesn't stand out as much so we don't even notice.  However, we would typically have:
- users (a list)
- user_count (the number of users)